src/ROUTES.js

```diff
@@ -21,9 +21,11 @@ export default {
     REPORT,
     REPORT_WITH_ID: 'r/:reportID',
     getReportRoute: reportID => `r/${reportID}`,
+    IOU_BILL_CURRENCY: 'iou/s/currency',
+    IOU_REQUEST_CURRENCY: 'iou/r/currency',
```

Comment:
What is `s` and `r`? Is it possible to match the existing routes? For example: `iou/request/${reportID}/currency`